how-to/editors-guide.md

```diff
@@ -188,7 +208,7 @@ Diversity is core to the pyOpenSci mission. As such it's important to have an
 editorial team comprised of an editor + 2 reviewers from diverse backgrounds.
 
 In your search for reviewers, please ensure ensure that there is diversity
-in the team supporting package review. [Specifically both reviewers should have different backgrounds and different gender-identities whenever possible](reviewer-diversity). pyOpenSci [supports mentoring new reviewers if needed!](review-mentorshipreview-mentorship)
+in the team supporting package review. Specifically both reviewers should have [different backgrounds and different gender-identities](reviewer-diversity) whenever possible. pyOpenSci [supports mentoring new reviewers if needed!](review-mentorship)
```

Comment:
not related to this line, but the line above has `please ensure ensure`, it should probably be just `please ensure`.
maybe I should create an issue instead of adding this note here XD